CONTRIBUTING.md

```diff
@@ -231,7 +249,7 @@ Note that `.` from abbreviations, such as `i.e.`, count.
 The `$` character may be used in mixins to mark a semantic separation in the name, in other words it allows to separate
 the actual name of the variable and the namespace.
 
-`@Unique` fields must be prefixed with `quilt$`, but `@Unique` methods don't need prefixes.
+All mixin fields and methods must be prefixed with `quilt$` in order to aid with debugging.
```

Comment:
Maybe extend this clause to also include duck interfaces (not injected interfaces)?